triveni-codev/GBWebApp-dev
# Darlp networks service descriptor 1203527428612576

Hi @triveni-codev @chriscodev @pauljustinhernandez 

Kindly check this PR for Service Descriptors. This is only for ATSC Cable and ATSC Terrestrial Types (don't test the ATSC 3)

ASANA: https://app.asana.com/0/1201943683850147/1203527428612576

TO TEST:
-logics on toggling valid/invalid values
-lazy loading implementation (shouldn't be loaded unless if modal is called)
-logic for extended channel name (once added, you can't add another instance) 
-logic for redistribution control descriptor type (once added, you can't add another instance also can't be updated)
-draft values being closed/cancelled should be good including the parent set of modals
-adding of one or more descriptor types (dirty change)
-updating of one or more descriptor types (dirty change) 
-retrieval of existing data from server is already displayed properly and can be added or updated with one or more descriptor types
-big commit for ADDING SERVICE DESCRIPTOR


DON'T TEST: Big Commit for UPDATING SERVICE DESCRIPTOR
